source/_integrations/tuya.markdown

```diff
@@ -39,67 +35,98 @@ The Tuya integration allows integrating all Powered by Tuya devices you have add
 
 ### Prerequisites
 
-Your devices need to first be added in the [Tuya Smart or Smart Life app](https://developer.tuya.com/en/docs/iot/tuya-smart-app-smart-life-app-advantages?id=K989rqa49rluq#title-1-Download).
-
-You will also need to create an account in the [Tuya IoT Platform](https://iot.tuya.com/).
+- Your devices need to first be added in the [Tuya Smart or Smart Life app](https://developer.tuya.com/en/docs/iot/tuya-smart-app-smart-life-app-advantages?id=K989rqa49rluq#title-1-Download).
+- You will also need to create an account in the [Tuya IoT Platform](https://iot.tuya.com/).
 This is a separate account to the one you made for the app. You cannot log in with your app's credentials.
 
-#### Create a project
+### Create a project
 
-**1.** Log in to the [Tuya IoT Platform](https://iot.tuya.com/).
+1. Log in to the [Tuya IoT Platform](https://iot.tuya.com/).
+2. In the left navigation bar, click `Cloud` > `Development`. If this is your first time setting up an account you will be asked to subscribe to a Cloud Development Plan. You can select and "purchase" the Trial Edition.
+3. On the page that appears, click `Create Cloud Project`.
+4. In the `Create Project` dialog box, configure `Project Name`, `Description`, `Industry`, and `Data Center`. For the `Development Method` field, select `Smart Home` from the dropdown list. For the `Data Center` field, select the zone you are located in.
+  ![](/images/integrations/tuya/image_001.png)
+5. Click `Create` to continue with project configuration.
+6. In Configuration Wizard, select the `Device Status Notification` API. `Smart Home` API products should already be selected by default.
+  ![](/images/integrations/tuya/image_002.png)
+7. Click `Authorize`.
 
-**2.** In the left navigation bar, click `Cloud` > `Development`. If this is your first time setting up an account you will be asked to subscribe to a Cloud Development Plan. You can select and "purchase" the Trial Edition.
+### Link devices by app account
 
-**3.** On the page that appears, click `Create Cloud Project`.
+1. Navigate to the `Devices` tab.
+2. Click `Link Tuya App Account` > `Add App Account`.
+  ![](/images/integrations/tuya/image_003.png)
+3. Scan the QR code that appears using the `Tuya Smart` app or `Smart Life` app.
+  ![](/images/integrations/tuya/image_004.png)
+4. Click `Confirm` on your `Tuya Smart` or `Smart Life` app.
+5. Navigate to the `All Devices` tab. Here you should be able to find the devices that have been added through your `Tuya Smart` or `Smart Life` account, as a confirmation everything worked.
 
-**4.** In the `Create Project` dialog box, configure `Project Name`, `Description`, `Industry`, and `Data Center`. For the `Development Method` field, select `Smart Home` from the dropdown list. For the `Data Center` field, select the zone you are located in.
+![](/images/integrations/tuya/image_005.png)
 
-![](/images/integrations/tuya/image_001.png)
+### Get authorization key
```

Comment:
Getting the authorisation key should be last thing to do in IoT as it's one step before going over to HA to add the integration. Rather than in between and say 'remember this for later'